buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -25,14 +38,27 @@ object Deps {
         val runtime = "org.antlr:antlr4-runtime:${Versions.antlr}"
     }
 
-    val z3 = "lib/com.microsoft.z3.jar"
-    val z3legacy = "lib/com.microsoft.z3legacy.jar"
+    val mpfr_java = when(getOs()) {
+        "windows" -> listOf("lib/mpfr_java-1.0.jar", "lib/mpfr_java-1.0-windows64.jar")
```

Comment:
Do we know that MPFR 1.0 and 1.4 are compatible on their binary interface? Can the newer version be built on Windows?